feature/xhr/instrument/index.js

```diff
@@ -298,6 +298,7 @@ ee.on('fetch-done', function (err, res) {
 
 // Create report for XHR request that has finished
 function end (xhr) {
+  this.endTime = this.endTime || loader.now()
```

Comment:
This was added to mimic the functionality seen in `fetch-done` seen (here)[https://github.com/newrelic/newrelic-browser-agent/blob/2c22571e5e260512000335f4926c417f295413f8/feature/xhr/instrument/index.js#L278]